MOBILE_CLIENT/COMMON/_HIGH/LEAKING_PERSONAL_INFORMATION/meta.json

```diff
@@ -2,7 +2,7 @@
   "risk_rating": "high",
   "short_description": "Personally Identifiable Information were detected on the system.",
   "references": {
-    "Practices for Protecting Electronic Restricted Data: A Quick Reference": "http://its.ucsc.edu/policies/rd.html",
+    "Practices for Protecting Electronic Restricted Data: A Quick Reference": "http://its.ucsc.edu/policies/rdpp.html",
```

Comment:
?

Reply:
I found out by mistake that this link was broken so I fix it